ui/app/helpers/all-features.js

```diff
@@ -0,0 +1,22 @@
+import { helper as buildHelper } from '@ember/component/helper';
+
+const ALL_FEATURES = [
+  'HSM',
+  'Performance Replication',
+  'DR Replication',
+  'MFA',
+  'Sentinel',
+  'AWS KMS Autounseal',
+  'GCP CKMS Autounseal',
+  'Seal Wrapping',
+  'Control Groups',
+  'Azure Key Vault Seal',
+  'Performance Standby',
+  'Namespaces',
+];
+
+export function allFeatures() {
+  return ALL_FEATURES;
```

Comment:
Could do `.slice(0)` here to make sure the list doesn't get mutated (but nbd either way - I don't think we're consistent about it in the other helpers).

Reply:
yeah I based this on other helpers so I'm going to keep it consistent for now.